### ISSUE TITLE
HTML formatter inserts a space between curly braces in regular text

### ISSUE THREAD

**Issue by ole** [biomejs/biome#11599]
When formatting this HTML:

```html
<p><code>struct MyView: View {}</code></p>
```

Biome insert a space between the curly braces:

```diff
- <p><code>struct MyView: View {}</code></p>
+ <p><code>struct MyView: View { }</code></p>
```

Is this intentional? I don't think a formatter should change the actual text of the page in such a way. If I don't want a space there in my code snippet, the formatter shouldn't insert one.

I tried with and without `html.experimentalFullSupportEnabled: true` and with and without `html.formatter.whitespaceSensitivity: "strict"`, but the result is always the same. Is there another config setting I'm overlooking?

Thanks!

### Environment information

<details><summary>Environment</summary>

```bash
biome rage --formatter
```

```txt
CLI:
  Version:                      2.5.11
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           macos

Environment:
  BIOME_DISTRIBUTION:                unset
  BIOME_LOG_FILE:                    unset
  BIOME_LOG_PATH:                    unset
  BIOME_LOG_PREFIX_NAME:             unset
  BIOME_LOG_LEVEL:                   unset
  BIOME_LOG_KIND:                    unset
  BIOME_CONFIG_PATH:                 unset
  BIOME_THREADS:                     unset
  BIOME_WATCHER_KIND:                unset
  BIOME_WATCHER_POLLING_INTERVAL:    unset
  BIOME_BINARY:                      unset
  RUST_BACKTRACE:                    unset
  NO_COLOR:                     unset
  TERM:                         xterm-256color
  JS_RUNTIME_VERSION:           unset
  JS_RUNTIME_NAME:              unset
  NODE_PACKAGE_MANAGER:         unset

Biome Configuration:
  Status:                       Loaded successfully.
  Path:                         biome.jsonc
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               true
  VCS enabled:                  false
  HTML full support enabled:    unset

Formatter:
  Format with errors:           true
  Indent style:                 unset
  Indent width:                 unset
  Line ending:                  unset
  Line width:                   unset
  Attribute position:           unset
  Bracket spacing:              unset
  Includes:                     unset

JavaScript Formatter:
  Enabled:                      unset
  JSX quote style:              unset
  Quote properties:             unset
  Trailing commas:              unset
  Semicolons:                   unset
  Arrow parentheses:            unset
  Bracket spacing:              unset
  Bracket same line:            unset
  Quote style:                  unset
  Indent style:                 unset
  Indent width:                 unset
  Line ending:                  unset
  Line width:                   unset
  Attribute position:           unset

JSON Formatter:
  Enabled:                      unset
  Indent style:                 unset
  Indent width:                 unset
  Line ending:                  unset
  Line width:                   unset
  Trailing Commas:              unset
  Expand lists:                 unset

CSS Formatter:
  Enabled:                      unset
  Indent style:                 unset
  Indent width:                 unset
  Line ending:                  unset
  Line width:                   unset
  Quote style:                  unset

GraphQL Formatter:
  Enabled:                      unset
  Indent style:                 unset
  Indent width:                 unset
  Line ending:                  unset
  Line width:                   unset
  Bracket spacing:              unset
  Quote style:                  unset

Workspace:
  Open Documents:               0
```

</details>

### Configuration

```JSON
{
  "$schema": "https://biomejs.dev/schemas/2.5.11/schema.json",
  "formatter": {
    "enabled": true,
  },
  "html": {
    // I tried with and without experimentalFullSupportEnabled
    // "experimentalFullSupportEnabled": true,
    "formatter": {
      "enabled": true,
      // I tried with and without whitespaceSensitivity: strict
      "whitespaceSensitivity": "strict"
    },
  }
}
```

### Playground link

https://biomejs.dev/playground/?bracketSameLine=true&enabledLinting=false&tab=formatter&pane=Diagnostics#files.main.html=PAAhAGQAbwBjAHQAeQBwAGUAIABoAHQAbQBsAD4ACgA8AGgAdABtAGwAPgAKAAkAPABiAG8AZAB5AD4ACgAJAAkAPABwAD4APABjAG8AZABlAD4AcwB0AHIAdQBjAHQAIABNAHkAVgBpAGUAdwA6ACAAVgBpAGUAdwAgAHsAfQA8AC8AYwBvAGQAZQA%2BADwALwBwAD4ACgAJADwALwBiAG8AZAB5AD4ACgA8AC8AaAB0AG0AbAA%2BAAoA

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by dyc3** [comment]
Nothing you're missing, its a bug. thanks for reporting.

**Comment by ole** [comment]
Thanks for the quick confirmation!

**Comment by dyc3** [comment]
I _thought_ I had already fixed this, and I was correct. It was fixed in #11474 , which has not been released yet. It'll be in the next patch release, which should be relatively soon (no exact ETA).

**Comment by ole** [comment]
@dyc3 Oh, fantastic. Thank you!

**Comment by ole** [comment]
Confirmed this is fixed in Biome 2.5.12. Thanks again to everyone involved.
